lib/validate.js

```diff
@@ -81,10 +81,27 @@ function validateRendition(rendition, index, location){
             }
         }
     }
+    if (rendition.watermark && rendition.watermark.watermarkContent !== undefined && rendition.watermark.watermarkContent !== null) {
+        validateWatermark(rendition.watermark);
+    }
 }
 
+function validateWatermark (paramsWatermark) {
```

Comment:
Actually we don't need any validation here in worker, since that is too late and has no real benefit in production since it happens asynchronous. It all should be in api-process with immediate error feedback.

The existing validate code here can probably mostly be removed I think, we just didn't tackle that yet.